packages/babel-preset-react-app/index.js

```diff
@@ -115,6 +115,14 @@ module.exports = function(api, opts) {
           regenerator: true,
         },
       ],
+      [
```

Comment:
This should probably have an opt-out flag in options. Just like the flow flag we now have.